### PR TITLE
Edit readme.md to cover the five new server level resources

### PR DESCRIPTION
**Pull request by hahndorf**
Made edits to readme.md to cover the changes from my previous pull request #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +94,10 @@ Currently, only FastCgiModule is supported.
 
 ## Versions
 
+### 1.7.0.0
```
Comment (tysonjhayes) [inline README.md]: This should probably be changed to ### Unreleased per @vors [comment](https://github.com/PowerShell/xDSCResourceDesigner/pull/20#issuecomment-114978784)

**Comment by tysonjhayes** [comment]
Looks good to me. :+1:

**Comment by TravisEz13** [comment]
Looks good, thanks

**Comment by hahndorf** [comment]
@TravisEz13 - so you are saying this looks good, but you never merged this. The 5 new resources were merged into dev, so I guess you should update the readme as well.

**Comment by TravisEz13** [comment]
I double checked at it was merged... re-closing.

**Comment by hahndorf** [comment]
Sorry Travis, all I can see is that the current dev branch readme.md doesn't have my changes and nowhere can I see that pull request #28 was merged. It also says here, `Closed with unmerged commits`. What am I missing?

**Comment by KarolKaczmarek** [comment]
Indeed, It looks like it haven't been merged.
After reopening, I can see there are some conflicts which may have prevented it from merging.
@hahndorf  Could you please resolve them?
```
Step 1: From your project repository, check out a new branch and test the changes.

git checkout -b hahndorf-ServerLevelResReadMe dev
git pull https://github.com/hahndorf/xWebAdministration.git ServerLevelResReadMe

Step 2: Merge the changes and update on GitHub.

git checkout dev
git merge --no-ff hahndorf-ServerLevelResReadMe
git push origin dev
```

**Comment by tysonjhayes** [comment]
@hahndorf looks like this is still a bad merge, do you want/need some assistance in trying to get this merged correctly? I don't have a ton of experience with it but I'd be happy to help.

**Comment by tysonjhayes** [comment]
Hi @hahndorf - are you still working on this? It's been a while since we've seen any activity on this. If you are not working on this anymore (or I don't hear from you) I'm going to fork this and try to fix it up myself as we'd really like some of this functionality. Please let me know.

**Comment by hahndorf** [comment]
Hi @tysonjhayes, sorry, I haven't had access to a safe PC in months, so please go ahead and try to fix it yourself. There are just a few new lines in the readme that describe the new features I added.  
I'm backpacking in Asia for the last 16 months without my own PC, so I'm very unreliable while on the road.

**Comment by tysonjhayes** [comment]
Sounds like a great time! I'll leave this open for now as reminder to myself to get it updated. Thanks for submitting it in the first place!

**Pull request closed without merging**